Source/WebKit/UIProcess/XR/ios/WKARPresentationSession.h

```diff
@@ -48,6 +48,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, nonnull, retain, readonly) id<MTLSharedEvent> completionEvent;
 @property (nonatomic, nullable, retain, readonly) id<MTLTexture> colorTexture;
 @property (nonatomic, readonly) NSUInteger renderingFrameIndex;
+@property (nonatomic, readonly) bool sessionEndRequested;
```

Comment:
this is read on some queue, but written on the main thread, should this be an atomic property instead?